### PR TITLE
Improve docs: Getting started dev & README (SHOOP-996)

### PR DESCRIPTION
**Pull request by suutari-ai**

### CODE REVIEW

File: doc/getting_started_dev.rst [hunk 1]
```diff
@@ -6,31 +6,39 @@ Getting Started with Shoop Development
...
+Github fork of Shoop for creating pull requests.  Github pull requests
+are used to get your changes into Shoop Base.
 
-.. code-block:: shell
-
-   pip install -e .[testing]
+ 1. If you haven't done so already, create a fork of Shoop in Github by
+    clicking the "Fork" button at https://github.com/shoopio/shoop and
+    clone the fork to your computer as usual. See `Github Help about
+    Fork A Repo <https://help.github.com/articles/fork-a-repo/>`__ for
```
Comment (akx) [inline doc/getting_started_dev.rst]: > s/about Fork A Repo/about forking repos/

File: doc/getting_started_dev.rst [hunk 2]
```diff
@@ -6,31 +6,39 @@ Getting Started with Shoop Development
...
+    details.
 
-If you don't need the testing dependencies, drop the ``[testing]`` suffix.
-Note: Some tests might need even more dependencies, then try
-``[everything]`` suffix.
-
-.. note::
+ 2. Setup a virtualenv and activate it.  You may use the traditional
+    ``virtualenv`` command, or newer ``python -m venv`` if you're using
```
Comment (akx) [inline doc/getting_started_dev.rst]: > s/newer/the newer/

File: doc/getting_started_dev.rst [hunk 3]
```diff
@@ -6,31 +6,39 @@ Getting Started with Shoop Development
...
 
-   If you get the following error::
+    .. code-block:: shell
 
-     --editable=.[testing] should be formatted with svn+URL, ...
+       virtualenv shoop-venv
+       . shoop-venv/bin/activate
 
-   you should upgrade your pip:
+ 3. Finaly, you'll need to install Shoop to the activated virtualenv in
```
Comment (akx) [inline doc/getting_started_dev.rst]: > Finally, you'll need to install Shoop in the activated virtualenv in

File: doc/getting_started_dev.rst [hunk 4]
```diff
@@ -6,31 +6,39 @@ Getting Started with Shoop Development
...
+    development mode.  To do that, run the following commands in the root of
+    the checkout (within activated virtualenv):
```
Comment (akx) [inline doc/getting_started_dev.rst]: > s/within/within the/

**Comment by suutari-ai** [comment]
Now?

**Comment by akx** [comment]
Now!

**Pull request merged**